spectral/scripts/rules.sh

```diff
@@ -4,10 +4,33 @@
 ## This apis can be used for testing
 ## Script needs to run in the test working directory
 
+export BASE_URL='https://raw.githubusercontent.com/redhat-developer/app-services-sdk-js/main/.openapi/'
 
 echo "fetching kas-fleet-manager from main" 
-wget -P .openapi https://raw.githubusercontent.com/redhat-developer/app-services-sdk-js/main/.openapi/kas-fleet-manager.yaml
+wget -P .openapi ${BASE_URL}kas-fleet-manager.yaml
+echo "linting kas-fleet-manager OAS file" 
+yarn spectral lint .openapi/kas-fleet-manager.yaml -v -r ./examples/.spectral-local.yaml
```

Comment:
```suggestion
spectralConfig= ./examples/.spectral-local.yaml
baseUrl='https://raw.githubusercontent.com/redhat-developer/app-services-sdk-js/main/.openapi/'
folder=".openapi"
file=kas-fleet-manager.yaml
wget -P $folder ${baseUrl}${file}
echo "linting $file OAS file" 
yarn spectral lint $folder/$file-v -r $spectralConfig
```

Reply:
`export` is global whilst not using the `export` and just assigning is just local. ok got ya. Will remember that.